proc/proc_test.go

```diff
@@ -495,6 +495,34 @@ func TestSwitchThread(t *testing.T) {
 	})
 }
 
+func TestCGONext(t *testing.T) {
```

Comment:
Does this test fail without your patch? I think there is already a CGO next test (I know it was my suggestion to write this test), but I want to make sure we have a test that triggers the behavior.